dluhc-component-library/src/components/siteSelectionForm/components/BooleanInput.tsx

```diff
@@ -0,0 +1,21 @@
+interface BooleanInputProps {
+  value: boolean;
+  onChange: (value: boolean) => void;
+}
+
+const BooleanInput = ({ value, onChange }: BooleanInputProps) => {

```

Comment:
`Non-blocker` I meant this should be called "Checkbox" in my previous comment but this is fine